src/solvers/refinement/string_constraint_generator_float.cpp

```diff
@@ -201,7 +201,8 @@ string_exprt string_constraint_generatort::add_axioms_for_string_of_float(
   mod_exprt integer_part(round_expr_to_zero(f), max_non_exponent_notation);
   // We should not need more than 8 characters to represent the integer
   // part of the float.
-  string_exprt integer_part_str=add_axioms_from_int(integer_part, 8, ref_type);
+  string_exprt integer_part_str=add_axioms_from_int(
+    integer_part, from_integer(10, integer_part.type()), ref_type, 8);
```

Comment:
When param lists are too long like this, put one param per line (like the other `add_axioms_from_int` call below)